charts/function-mesh-operator/README.md

```diff
@@ -0,0 +1,188 @@
+
+It is recommended to use [cert-manager](https://cert-manager.io/) to manage these certificates and you can install the cert-manager as follows.
+
+```shell
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm install \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --version v1.8.0 \
+  --set installCRDs=true
+```
+
+### Install Function Mesh
+
+> **Note**
+>
+> - Before installation, ensure that Helm v3 is installed properly.
+> - For the use of `kubectl` commands, see [kubectl command reference](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands).
+
+1. Add the StreamNative Function Mesh repository.
+
+    ```shell
+    helm repo add function-mesh http://charts.functionmesh.io/
+    helm repo update
+    ```
+
+2. Install the Function Mesh Operator.
+
+    Let's set some variables for convenient use later.
+
+    ```shell
+    export FUNCTION_MESH_RELEASE_NAME=function-mesh  # change the release name according to your scenario
+    export FUNCTION_MESH_RELEASE_NAMESPACE=function-mesh  # change the namespace to where you want to install Function Mesh
+    ```
+
+    Install the Function Mesh Operator via following command.
+
+    > **Note**
+    >
+    > - If no Kubernetes namespace is specified, the `default` namespace is used.
+    >
+    > - If the namespace ${FUNCTION_MESH_RELEASE_NAMESPACE} doesn't exist yet, you can add the parameter `--create-namespace ` to create it automatically.
+
+    ```shell
+    helm install ${FUNCTION_MESH_RELEASE_NAME} function-mesh/function-mesh-operator -n ${FUNCTION_MESH_RELEASE_NAMESPACE}
+    ```
+
+    You can refer to the [Values](#values) table above to learn about the configurable parameters of the Function Mesh operator and their default values.
+
+    For example, if you want to enable `pprof` for the Function Mesh Operator, set the `controllerManager.pprof.enable` to `true`.
+    
+    ```shell
+    helm install ${FUNCTION_MESH_RELEASE_NAME} function-mesh/function-mesh-operator -n ${FUNCTION_MESH_RELEASE_NAMESPACE} \
+      --set controllerManager.pprof.enable=true
+    ```
+    
+3. Check whether Function Mesh is installed successfully.
+
+    ```shell
+    kubectl get pods --namespace ${FUNCTION_MESH_RELEASE_NAMESPACE} -l app.kubernetes.io/instance=function-mesh
+    ```
+
+    **Output**
+
+    ```
+    NAME                                               READY   STATUS    RESTARTS   AGE
+    function-mesh-controller-manager-5f867557c-d6vf4   1/1     Running   0          8s
+    ```
+
+### Verify installation
+
+- This example shows how to verify whether Function Mesh is installed successfully.
```

Comment:
I think this step is repeated with step 3 in the previous section. You can remove either one of them.